io/net/htx_nic_devices.py

```diff
@@ -71,20 +71,23 @@ def setUp(self):
                     networkinterface.save(ipaddr, self.netmask)
                 networkinterface.bring_up()
         self.host_distro = distro.detect()
```

Comment:
There is no need of self here for `self.host_distro` this variable is not used later.